### PR TITLE
[LM-13211] Zoom image

### PR DESCRIPTION
**Pull request by divyanshgandhilm**
## Description
Added support for zooming in all images. 

**Features**

**Bugs**

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Hotfix (fixes a breaking changes)
- [x] New Feature (non-breaking change which adds functionality)
- [ ] Other, please describe:

### Author Checklist

- [ ] Assignee added
- [ ] Reviewer added
- [ ] Description added
- [ ] Unit test cases added

### Reviewer Checklist

- [ ] PR title starts with Jira ticket number prefix
- [ ] If hotfix, Add Hotfix as prefix in PR title
- [ ] Label added
- [ ] Commit messages follows standard format (added, changed, removed, code commented and chore)
- [ ] Description about the Feature/ Bugs added

### CODE REVIEW

File: lib/packages/ui/lib/src/widgets/media/image.dart [hunk 1]
```diff
@@ -79,19 +82,15 @@ class _LMImageState extends State<LMChatImage> {
   }
 
   @override
-  void didUpdateWidget(covariant LMChatImage oldWidget) {
-    super.didUpdateWidget(oldWidget);
-    style = widget.style ?? LMChatTheme.theme.imageStyle;
+  void dispose() {
+    super.dispose();
```
Comment (AnujLM) [inline lib/packages/ui/lib/src/widgets/media/image.dart:86]: remove dispose() if we are not using it to dispose any other instance.

File: lib/packages/ui/lib/src/widgets/media/image.dart [hunk 2]
```diff
@@ -79,19 +82,15 @@ class _LMImageState extends State<LMChatImage> {
...
   @override
   Widget build(BuildContext context) {
-    return AbsorbPointer(
-      absorbing: widget.onTap == null,
-      child: GestureDetector(
-        onTap: () => widget.onTap?.call(),
-        child: _buildImageWidget(),
-      ),
+    return GestureDetector(
+      onTap: () => widget.onTap?.call(),
```
Comment (AnujLM) [inline lib/packages/ui/lib/src/widgets/media/image.dart:92]: can we change it like 
`onTap: widget.onTap,`

File: lib/packages/ui/lib/src/widgets/media/image.dart [hunk 3]
```diff
@@ -108,6 +107,30 @@ class _LMImageState extends State<LMChatImage> {
     return const SizedBox();
   }
 
+  Widget _buildPhotoView({
```
Comment (AnujLM) [inline lib/packages/ui/lib/src/widgets/media/image.dart:110]: add comments for this function

**Review by AnujLM — APPROVED** [review]

File: lib/packages/ui/lib/src/widgets/media/image.dart (hunk 1 above)
Comment (divyanshgandhilm) [reply lib/packages/ui/lib/src/widgets/media/image.dart]: Done

File: lib/packages/ui/lib/src/widgets/media/image.dart (hunk 2 above)
Comment (divyanshgandhilm) [reply lib/packages/ui/lib/src/widgets/media/image.dart]: Done

File: lib/packages/ui/lib/src/widgets/media/image.dart (hunk 3 above)
Comment (divyanshgandhilm) [reply lib/packages/ui/lib/src/widgets/media/image.dart:119]: Added in all functions

**Pull request merged**